docs/api/telegram.rst

```diff
@@ -60,6 +60,12 @@ about its business.
 
       *This attribute can be None if it's not provided by Telegram.*
 
+   .. py:attribute:: lang
+
+      The user's language, given by Telegram.
```

Comment:
Let's wait for the docs to see from where Telegram gets the language (client lang, os lang...).

Reply:
The upstream Telegram documentation now documents `language_code`, but does not specify where it comes from.